.github/workflows/app_release.yaml

```diff
@@ -0,0 +1,102 @@
+name: App Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      sem-ver:
+        required: true
+        description: 'Semver bump'
+        type: choice
+        default: minor
+        options:
+          - major
+          - minor
+          - patch
+  push:
+    branches:
+      - main
+    paths:
+      - "**"
+      - "!infra/**"
+      - "!.pre-commit-config.yaml"
+      - "!.terraform-version"
+      - "!CODEOWNERS"
+      - "!README.md"
+
+env:
+  RELEASE_BRANCH: main
+
+jobs:
+  prod_release:
+    uses: pagopa/dx/.github/workflows/legacy_deploy_pipelines.yaml@main
+    name: Code Review
+    secrets: inherit
```

Comment:
`legacy_deploy_pipelines` template uses ARM tokens that I don't see in this repo settings. Do you plan to also add a managed identity and a dedicated environment to this project? 
IMO, in the future plan of migrating this repo to `io-auth-n-identity-domain` we should create the managed identity there and make a copy for the secrets in this project, facilitating the future migration.

Reply:
You're right, I've created a PR in `io-auth-n-identity-domain` to create new identities to use in this repo as well: https://github.com/pagopa/io-auth-n-identity-domain/pull/111